urlparse4/cgurl.pyx

```diff
@@ -177,13 +214,22 @@ def unicode_handling(str):
     return bytes_str
 
 def urlsplit(url):
+    """
+    Use GURL to split the url. Needs to be reviewed!
```

Comment:
> Needs to be reviewed!

Do you have any particular concerns? Or do you mean the Cython implementation?

Reply:
hmm not at the moment since this is the old comment and we have already discussed this :)